helm/csi-isilon/k8s-1.21-values.yaml

```diff
@@ -0,0 +1,24 @@
+kubeversion: "v1.21"
+# IT IS RECOMMENDED YOU DO NOT CHANGE THE IMAGES TO BE DOWNLOADED.
+images:
+  # "images.attacher" defines the container images used for the csi attacher
+  # container.
+  attacher: k8s.gcr.io/sig-storage/csi-attacher:v3.4.0
+
+  # "images.provisioner" defines the container images used for the csi provisioner
+  # container.
+  provisioner: k8s.gcr.io/sig-storage/csi-provisioner:v3.1.0
+
+  # "images.snapshotter" defines the container image used for the csi snapshotter
+  snapshotter: k8s.gcr.io/sig-storage/csi-snapshotter:v5.0.1
+
+  # "images.registrar" defines the container images used for the csi registrar
+  # container.
+  registrar: k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.5.1
+
+  # "images.resizer" defines the container images used for the csi resizer
+  # container.
+  resizer: k8s.gcr.io/sig-storage/csi-resizer:v1.4.0
+
+  # "images.externalhealthmonitorcontroller" defines the container images used for the csi external health monitor controller
+  # container.
```

Comment:
Missed this line at the end.
externalhealthmonitorcontroller: k8s.gcr.io/sig-storage/csi-external-health-monitor-controller:v0.5.0

Reply:
addressed